tests/unit_tests/data_model/test_format_checkers.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/python3
+
+import pytest
+
+from simtools.data_model import format_checkers
+
+
+def test_check_astropy_unit():
+    assert format_checkers.check_astropy_unit("dimensionless")
+    assert format_checkers.check_astropy_unit("m")
```

Comment:
How about testing also the case in which it fails?

Reply:
Added two tests for this.

Reply:
Thanks!